airbyte-integrations/connectors/source-surveymonkey/pyproject.toml

```diff
@@ -17,7 +17,7 @@ include = "source_surveymonkey"
 
 [tool.poetry.dependencies]
 python = "^3.9,<3.12"
-airbyte-cdk = "==0.51.40"
+airbyte-cdk = "==0.62.0"
 vcrpy = "==4.1.1"
 urllib3 = "==1.26.18"
```

Comment:
I think we can get rid of these dependencies.

Reply:
Updated